Android/Google.Android.DataTransport/build.cake

```diff
@@ -7,21 +7,19 @@
 
 var TARGET = Argument ("t", Argument ("target", "ci"));
 
-string v="2.2.0";
-
 Dictionary<string, string> URLS_ARTIFACT_FILES= new Dictionary<string, string>()
 {
 	{
-		$"https://maven.google.com//com/google/android/datatransport/transport-api/{v}/transport-api-{v}.aar",
-		$"./externals/android/transport-api-{v}.aar"
+		$"https://maven.google.com//com/google/android/datatransport/transport-api/2.2.0/transport-api-2.2.0.aar",
+		$"./externals/android/transport-api-2.2.0.aar"
```

Comment:
All the other versions are `2.2.2`, should this one be `2.2.0`?

Reply:
yes. that is/was the proble for this PR. Google started publishing out of sync